nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-frontend/src/main/nifi/src/app/ui/common/controller-service/disable-controller-service/disable-controller-service.component.html

```diff
@@ -33,11 +33,25 @@ <h2 mat-dialog-title>Disable Controller Service</h2>
                             </div>
                             <div class="flex flex-col">
                                 <div>Scope</div>
-                                <div class="accent-color font-medium">Service and referencing components</div>
+                                <div class="accent-color font-medium">
+                                    Service and referencing components
+                                    <i
+                                        class="fa fa-info-circle"
+                                        nifiTooltip
+                                        [tooltipComponentType]="TextTip"
+                                        tooltipInputData="Referencing components must be disabled/stopped in order to disable this service."></i>
+                                </div>
```

Comment:
This one is weird since the info icon is in the "value" area and not the label.
<img width="295" alt="Screenshot 2024-04-24 at 3 36 51 PM" src="https://github.com/apache/nifi/assets/713866/bf0cccad-923a-4461-ae81-b146564443f3">

Thoughts on moving to something like this for this case?
```suggestion
                                <div>
                                    <span class="accent-color font-medium">Service and referencing components</span>
                                    <i
                                        class="fa fa-info-circle ml-1"
                                        nifiTooltip
                                        [tooltipComponentType]="TextTip"
                                        tooltipInputData="Referencing components must be disabled/stopped in order to disable this service."></i>
                                </div>

```
<img width="274" alt="Screenshot 2024-04-24 at 3 36 23 PM" src="https://github.com/apache/nifi/assets/713866/f4156d5e-ac41-4586-a3a3-54785db5d8c1">